crates/utils/re_smart_channel/src/lib.rs

```diff
@@ -19,6 +19,7 @@ pub use sender::Sender;
 /// Identifies in what context this smart channel was created, and who/what is holding its
 /// receiving end.
 #[derive(Clone, Debug, PartialEq, Eq, Hash, serde::Deserialize, serde::Serialize)]
+#[expect(clippy::large_enum_variant)]
```

Comment:
Why is this large all of a sudden? Ah, because `re_uri`.

Reply:
yeah looks like I tripped a treshhold. Something in there should probably be a separate alloc, but didn't want to go there right now